Capri.Web/Configuration/Service/ServiceConfiguration.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.Extensions.DependencyInjection;
+using Capri.Services.Account;
+using Capri.Services.Token;
+using Capri.Services.Users;
+using Capri.Services.Proposals;
+using Capri.Services.Promoters;
+using Capri.Services.Courses;
+using Capri.Services.Faculties;
+using Capri.Services.Institutes;
+
+namespace Capri.Web.Configuration.Service
+{
+    public static class ServiceConfiguration
+    {
+        public static void AddServiceConfiguration(this IServiceCollection services)
```

Comment:
I would rename that method to `AddServicesConfiguration`. I think it better explains what's going on :)

Reply:
Done